### DISCUSSION TITLE
Examples from youtube videos

### DISCUSSION THREAD

**Discussion by stapperfend** [InteractiveComputerGraphics/SPlisHSPlasH#307] [General]
Are the examples from your youtube videos available somewhere? 
I am specifically interested in the Temperature-Dependent Viscosity example that is shown in the ["A Physically Consistent Implicit Viscosity Solver for SPH Fluids" Youtube video](https://www.youtube.com/watch?v=D_nEhix1G-w). 

I found some example scenes, but not these specific applications.

**Comment by janbender** [comment]
Unfortunately, we never integrated the temperature simulation in the main code of the project. Therefore, the temperature-dependent experiments are not available.

**Reply by stapperfend** [reply]
May I ask why you didn't integrate it in the main code? 
And are there any plans to integrate that in future versions?

**Reply by janbender** [reply]
Just because nobody had time to clean up the code and to write a user interface. Since we are again working with temperature solvers at the moment, probably we will integrate this in a future version.

**Reply by stapperfend** [reply]
Okay, that sounds good. 
I would love to implement that in a 3d printing simulation that I am currently working on. Looking forward to that in the future.

**Reply by JonathanBelnoue** [reply]
Hello is this integration still planned? This would be very useful indeed for manufacturing simulation more generally
